packages/cli-kit/src/public/node/ui.tsx

```diff
@@ -686,14 +686,18 @@ export const keypress = async (stdin = process.stdin, uiDebugOptions: UIDebugOpt
   })
 }
 
+export function isTTY(stdin: NodeJS.ReadStream | undefined = undefined, uiDebugOptions: UIDebugOptions = defaultUIDebugOptions) {
+  return uiDebugOptions.skipTTYCheck || stdin || terminalSupportsRawMode()
```

Comment:
We might want to wrap this with a `Boolean` so that we don't accidentally return one of these values.

```suggestion
  return Boolean(uiDebugOptions.skipTTYCheck || stdin || terminalSupportsRawMode())
```